docs/software/media.md

```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
 
 Source code: <https://github.com/badaix/snapcast>
 
+## Navidrome
+
+An open source web-based music collection server and streamer, written in Go.  
+It is API compatible to the [Subsonic media player](http://www.subsonic.org/pages/index.jsp).
```

Comment:
```suggestion
It is compatible with the [Subsonic media player](http://www.subsonic.org/pages/index.jsp).
```

Reply:
Take over this change in the Releasenotes PR.